6238_prototype/src/main/java/frc/robot/RobotContainer.java

```diff
@@ -69,12 +69,11 @@ private void addBall() {
     ballSubsystem.setDefaultCommand(ballManualCommand);
 
     new JoystickButton(joystick, IOConstants.START_INTAKE)
-      .whenPressed(() -> ballManualCommand.startIntake())
-      .whenReleased(() -> ballManualCommand.stopIntake());
-  
+      .whenPressed(new RunIntakeCommand(ballSubsystem))
+      .whenReleased(new RetractIntakeCommand(ballSubsystem));
+
     new JoystickButton(joystick, IOConstants.START_SHOOTER)
-      .whenPressed(() -> ballManualCommand.startShooter())
-      .whenReleased(() -> ballManualCommand.motorOff());
+      .whenPressed(new ShooterCommand(ballSubsystem));
```

Comment:
We should have a second command that runs `whenReleased` that stops the shooter.  As currently written the shooter will keep running indefinitely.

Reply:
You can, but you still need a `whenReleased` trigger to cause the shooter command to stop running.